### ISSUE TITLE
Zotero sync to notion issue 

### ISSUE THREAD

**Issue by Osoreski** [dvanoni/notero#389]
Hello, I am contacting you because I am experiencing an issue. I think it might be related to the synchronization between Zotero and the database. Indeed, nothing happens in Zotero after configuring Notero, and I don't see any file loading messages. Similarly, when I click on "Sync item to Notion," nothing happens. I have tried to perform the operation several times, uninstalled and reinstalled Zotero, but nothing has changed. I am French, so I apologize if some parts are not very understandable. I wonder if this could be because Zotero is configured in French and Notero in English.
Thank you for your response

<img width="577" alt="Capture d’écran 2023-12-09 à 17 04 57" src="https://github.com/dvanoni/notero/assets/153367246/b64fe740-e5e9-41ef-9b86-96e019afb4c0">
<img width="1000" alt="Capture d’écran 2023-12-09 à 17 06 40" src="https://github.com/dvanoni/notero/assets/153367246/7679dc5a-6f54-4492-9d5e-4931e9667ce1">

**Comment by dvanoni** [comment]
Hi @Osoreski, thanks for reporting this and sending those screenshots. Your message was very clear!

Everything looks to be configured correctly, so I'm not sure why Notero isn't working as expected. 🤔 

Can you send me a couple screenshots of the Notero logs by following these steps?

1. Open the error console from the menu: Outils > Développeur > Error Console
2. Scroll to the bottom of the error console (there should be some log messages from Notero)
3. Take a screenshot of the error console
4. At the top of the error console, click the "Clear" button
5. Try syncing an item to Notion
6. Take a screenshot of the error console

Thank you!

**Comment by dorothy110** [comment]
![ERRROR](https://github.com/dvanoni/notero/assets/83738804/75e81060-7c1c-406b-bc73-ced2935e187e)
Hi @dvanoni , Thanks for your tools! I downloaded it today and facing the same issue, here is the screenshot screenshot of the error console.

**Comment by dvanoni** [comment]
Hi @dorothy110, thanks for sending the screenshot! Unfortunately, I don't see anything related to Notero in there. Would you mind following steps 4–6 in my message above and sending another screenshot?

**Comment by dorothy110** [comment]
![image](https://github.com/dvanoni/notero/assets/83738804/8b5043e1-1796-4c9a-85ee-3080905782b9)
Hi @dvanoni , Thanks again for your help! I restarted Zotero and tried syncing again today, and it looks like all the problems were automatically solved. I apologize for the inconvenience I caused.

For anyone who might have the same issue as me, here is my experience:
After entering the database ID from Notion, I restarted Zotero. Even though at that time notes were not synced, just give it some time to wait. I checked Zotero today, but it might already have been synced automatically earlier.
